### PR TITLE
Fix Register route inside the login page

### PR DESCRIPTION
**Pull request by alessandromontividiu03**
fix: https://github.com/nopSolutions/nopCommerce/issues/4181

### CODE REVIEW

File: src/Presentation/Nop.Web.Framework/Mvc/Filters/HttpsRequirementAttribute.cs [hunk 1]
```diff
@@ -119,6 +119,10 @@ public void OnAuthorization(AuthorizationFilterContext filterContext)
                 if (!DataSettingsManager.DatabaseIsInstalled)
                     return;
 
+                //ignore this rule for localhost
+                if (_webHelper.IsLocalRequest(filterContext.HttpContext.Request))
+                    return;
```
Comment (DmitriyKulagin) [inline src/Presentation/Nop.Web.Framework/Mvc/Filters/HttpsRequirementAttribute.cs]: This code is not related to solving the problem. Please exclude it from the pull request.

Comment (alessandromontividiu03) [reply src/Presentation/Nop.Web.Framework/Mvc/Filters/HttpsRequirementAttribute.cs]: @DmitriyKulagin its done, thanks for reviewing it.

**Comment by DmitriyKulagin** [comment]
@alessandromontividiu03 Thanks.
